flink-examples/doc/flink-high-error-count-alert/filters/01-stdin-input.conf

```diff
@@ -0,0 +1,2 @@
+input { stdin { } }
```

Comment:
We add the license header to config files as well, see under `config/` in pravega.

Reply:
done.